frontends/web/package.json

```diff
@@ -4,8 +4,8 @@
   "version": "0.0.0",
   "scripts": {
     "gen-ts-css": "tcm src --camelCase dashes",
-    "build": "yarn run gen-ts-css && preact build --no-sw --no-esm --no-inline-css --no-prerender --verbose",
-    "dev": "yarn run gen-ts-css && preact watch",
+    "build": "yarn run gen-ts-css && preact build --no-sw --no-esm --no-inline-css --no-prerender --no-prerenderUrls --no-json --verbose",
```

Comment:
The commit says "to suppress the no prerender url json warning" ...what does that warning say and why?

Reply:
React apps can pre-render routes during the build step creating various HTML files, this is important for webapps, but does not matter for the BitBoxApp. To do so preact-cli by default expects a JSON file with the url's it should pre-render.

>     --prerenderUrls    Path to pre-rendered routes config  (default prerender-urls.json)

The warning is about that preact-cli cannot find the prerender-urls.json file.